assets/sass/_theme/sections/programs.sass

```diff
@@ -68,42 +68,40 @@ ol.programs
 
         .essential-container
             margin-top: $spacing-3
-            @include media-breakpoint-up(md)
+@include media-breakpoint-up(md)
```

Comment:
attention petite vérif à faire du sass

Reply:
I'm on it :)

Reply:
sorry je maitrise pas bien les suggestions